src/index.ts

```diff
@@ -1,2 +1,3 @@
-export * from "./greet.js";
-export * from "./types.js";
+#!/usr/bin/env node
+
+export { runCli as bin } from "./runCli.ts";
```

Comment:
I'm curious about the use case for the TS api.  Wouldn't anyone that might use this, be better off just using `package-json-validator` directly?

Reply:
Heh, yeah, probably. Once in a while I've found exporting a CLI function on its own to be helpful. I figured it couldn't hurt here 🤷 but am not passionate about it. What would you prefer?

Reply:
Also, it's easier to add than to remove. So maybe leave it out until there's a demand?